src/modules/users/application/use-cases/soft-delete-user.use-case.ts

```
@@ -0,0 +1,91 @@
+        userEmail: targetUser.email || '',
+        username: targetUser.username,
+      },
+      ipAddress,
+      userAgent,
+    });
+  }
+
+  private async isUserAdmin(user: User): Promise<boolean> {
+    const userWithRoles = await this.userRepository.findWithRoles(user.id);
+    if (!userWithRoles) return false;
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)